### PR TITLE
feat: add truncate_input_tokens parameter for embeddings

### PR DESCRIPTION
**Pull request by jezekra1**
Ref #272

### CODE REVIEW

File: src/genai/_generated/endpoints.py [hunk 1]
```diff
@@ -120,6 +120,36 @@ class RequestIdDeleteEndpoint(ApiEndpoint):
     version: str = "2023-11-22"
 
 
+class SystemPromptRetrieveEndpoint(ApiEndpoint):
```
Comment (jezekra1) [inline src/genai/_generated/endpoints.py:123]: Should we add a service for these endpoints?

Comment (Tomas2D) [reply src/genai/_generated/endpoints.py:123]: I think that they should not be even part of the public API.
Ensure that this PR contains changes related to embeddings only.

Comment (jezekra1) [reply src/genai/_generated/endpoints.py]: Ok, I removed other changes manually, we should try to keep the schemas up to date, but we can make another PR

**Review by Tomas2D — APPROVED** [review]
🚀

**Pull request merged**